package.nls.ko.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plantuml.exportCurrent.title": "현재 다이어그램 내보내기",
+	"plantuml.exportDocument.title": "현재 다이어그램 파일 내보내기",
+	"plantuml.exportWorkspace.title": "작업공간의 다이어그램 내보내기",
+	"plantuml.preview.title": "현재 다이어그램 미리보기",
+	"plantuml.URLCurrent.title": "현재 다이어그램에 대한 URL 생성하기",
+	"plantuml.URLDocument.title": "현재 다이어그램 파일에 대한 URL 생성하기",
+	"plantuml.extractSource.title": "그림에서 PlantUML 다이어그램 소스 추출하기",
+	"plantuml.configuration.configTitle": "PlantUML 환경설정",
+	"plantuml.configuration.exportFormat": "내보내기 형식. 매전 내보내기할 때마다 선택하려면 비워두십시오.",
```

Comment:
Typo
`매전 내보내기` -> `매번 내보내기`

But how about rephrase like this?
```
내보내기 형식. 매번 내보낼 때마다 선택하려면 비워두십시오.
```
Because `매번 내보내기할 때` seems weird to me.

Reply:
Thanks, fixed it.